src/views/authenticated/List.tsx

```diff
@@ -21,13 +21,23 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 			.sort(comparePurchaseUrgency);
 	}, [searchTerm, unfilteredListItems]);
 
+	const Header = () => {
+		return (
+			<p>
+				Hello from the <code>/list</code> page!
+			</p>
+		);
+	};
+
+	if (!listPath) {
+		return <Header />;
+	}
+
 	// Early return if the list is empty
 	if (unfilteredListItems.length === 0) {
 		return (
 			<>
-				<p>
-					Hello from the <code>/list</code> page!
-				</p>
+				<Header />
 				<section>
 					<h2>Your list is ready!</h2>
```

Comment:
I like the returning of the header if no list has any items. Can we get rid of the statement "Your list is ready" if this is the case too? :) I think it might be contradicting to tell users "your list is ready" if the list is empty!

Reply:
Done 🤝